README.md

```diff
@@ -48,6 +48,36 @@ gracefully (and without causing problems for others!).**
 The `[extras]` argument tells pip to install packages to fullfill the dependencies of the
 `ga4gh.vrs.extras` package.
 
+## Getting started with Docker
+
```

Comment:
This section is good, but should be integrated in the context of _why_ they would want to get started with Docker (answer: because the ga4gh.vrs.extra utils are dependent upon them).  I think this README would therefore make sense with a little transition text, in an introductory subsection under "Installing dependencies for ga4gh.vrs.extras" section, below.